### ISSUE TITLE
Updating from 0.9.8 to 1.0 breaks date format specifier %F

### ISSUE THREAD

**Issue by organman91** [allinurl/goaccess#434]
Hi all,

Hopefully I've just overlooked something in the changelog - let me know if the config looks wrong.

Here is the nginx log format:
```
log_format custom
    '$remote_addr "$ssl_client_s_dn" '
    '"$auth_remote_user" [$time_iso8601] '
    '"$request" $status $body_bytes_sent '
    '"$http_referer" "$http_user_agent" '
    '$request_time';
```

And this is the goaccess.conf that previously worked in 0.9.8 but no longer works in 1.0:
```
color_scheme 0
date_format %F
time_format %T
log_format %h "%^" "%^" [%dT%^] "%r" %s %b "%R" "%u" %T
```

**Comment by allinurl** [comment]
It looks like you are missing the `%t` specifier within the `log-format`. Assuming your dates look like `2006-01-02T15:04:05-07:00`, can you please try:

    date_format %F
    time_format %T
    log_format %h "%^" "%^" [%dT%t-%^] "%r" %s %b "%R" "%u" %T

**Comment by organman91** [comment]
It's actually +00:00 instead of -07:00 since we're on UTC instead of local time (e.g. `[2016-06-10T21:14:25+00:00]`) - using the following works on 0.9.8 but, again, not 1.0

```
date_format %F
time_format %T
log_format %h "%^" "%^" [%dT%t+%^] "%r" %s %b "%R" "%u" %T
```

This error, specifically:
```Fatal error has occurred
Error occured at: src/goaccess.c - main - 1223
Nothing valid to process. Verify your date/time/log format.```

**Comment by allinurl** [comment]
Can you please post a few lines from you log so I can take a look? Thanks

**Comment by organman91** [comment]
I can give you an example of what it would look like:

```
10.11.12.13 "-" "user@example.com" [2016-06-10T16:41:11+00:00] "GET /example HTTP/1.1" 200 12345 "https://example.com/origin-url" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_5) AppleWeb    Kit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.84 Safari/537.36" 0.053
```

Save for the user agent, these are obviously made-up values.

**Comment by allinurl** [comment]
The following works:

    goaccess -f log --log-format='%h %^[%dT%t+%^] "%r" %s %b "%R" "%u" %T' --date-format='%Y-%m-%d' --time-format='%T'

I'll have a fix for the `%F` specifier. Thanks for pointing it out.

**Comment by organman91** [comment]
Yay! That config works for me, thanks for the help.

**Comment by allinurl** [comment]
This has been fixed and just deployed on **[v1.0.1.](https://goaccess.io/download)**.

Thanks for reporting it!
